var/spack/repos/builtin/packages/snap/package.py

```diff
@@ -0,0 +1,53 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Snap(MakefilePackage):
+    """SNAP is a fast and accurate aligner for short DNA reads. It is
+       optimized for modern read lengths of 100 bases or higher, and takes
+       advantage of these reads to align data quickly through a hash-based
+       indexing scheme."""
+
+    homepage = "http://snap.cs.berkeley.edu/"
+    url      = "https://github.com/amplab/snap/archive/v1.0beta.18.tar.gz"
+
+    version('1.0beta.18', '41e595fffa482e9eda1c3f69fb5dedeb')
```

Comment:
Yeah... You can add both if you want. If you add `preferred=True` to the last release, it will choose that version by default. Up to you, since it's your package.